tests/components/ozw/test_services.py

```diff
@@ -2,61 +2,61 @@
 from .common import setup_ozw
 
 
-async def test_services(hass, lock_data, sent_messages, lock_msg, caplog):
+async def test_services(hass, light_data, sent_messages, light_msg, caplog):
     """Test services on lock."""
-    await setup_ozw(hass, fixture=lock_data)
+    await setup_ozw(hass, fixture=light_data)
 
     # Test set_config_parameter list by label
     await hass.services.async_call(
         "ozw",
         "set_config_parameter",
-        {"node_id": 10, "parameter": 1, "value": "Disabled"},
+        {"node_id": 39, "parameter": 1, "value": "Disable"},
         blocking=True,
     )
     assert len(sent_messages) == 1
     msg = sent_messages[0]
     assert msg["topic"] == "OpenZWave/1/command/setvalue/"
-    assert msg["payload"] == {"Value": 0, "ValueIDKey": 281475154706452}
+    assert msg["payload"] == {"Value": 0, "ValueIDKey": 281475641245716}
 
     # Test set_config_parameter list by index int
     await hass.services.async_call(
         "ozw",
         "set_config_parameter",
-        {"node_id": 10, "parameter": 1, "value": 0},
+        {"node_id": 39, "parameter": 1, "value": 1},
         blocking=True,
     )
     assert len(sent_messages) == 2
     msg = sent_messages[1]
     assert msg["topic"] == "OpenZWave/1/command/setvalue/"
-    assert msg["payload"] == {"Value": 0, "ValueIDKey": 281475154706452}
+    assert msg["payload"] == {"Value": 1, "ValueIDKey": 281475641245716}
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5985897)